CodeStarterCamp_Week2/main.swift

```diff
@@ -6,24 +6,38 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
-func generateLottoNumbers() -> Set<Int> {
+func generateLottoNumbers() -> String {
     var randomIntegers: Set<Int> = Set<Int>()
     
     repeat {
         let randomInteger: Int = Int.random(in: 1..<46)
         randomIntegers.insert(randomInteger)
     } while randomIntegers.count < 6
     
-    return randomIntegers
+    
+    return randomIntegers.map({String($0)}).joined(separator: " ,")
```

Comment:
```suggestion
    return randomIntegers.map({String($0)}).joined(separator: ", ")
```
기존의 함수를 재사용하지 않고 반환값을 바꾼 이유가 궁금합니다!

Reply:
옵셔널으로 구현하려고 하다보니 발생한 문제인데요.
옵셔널 사용 시 String 구문이 if let 에서 Set -> String 으로 치환하는 과정에서 오류가 발생했었습니다.
따라서 해당 함수 바꿨었습니다.